lib/components/ContactUsForm.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import {
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  HStack,
+  Input,
+  Center,
+  Box,
+  Button,
+  Textarea,
+} from '@chakra-ui/react';
+
+interface contactFormProps {}
+
+interface NameFormProps {
+  firstNameInput: string;
+  lastNameInput: string;
+  setFirstInput: any;
+  setLastInput: any;
+  isFirstNameMissing: boolean;
+  isLastNameMissing: boolean;
+}
+
+const NameForm: React.FC<NameFormProps> = ({
+  firstNameInput,
+  lastNameInput,
+  setFirstInput,
+  setLastInput,
+  isFirstNameMissing,
+  isLastNameMissing,
+}) => {
+  return (
+    <HStack>
+      <Box w="50%">
+        <FormLabel htmlFor="first_name">First Name</FormLabel>
+        <Input
+          id="first_name"
+          type="text"
+          value={firstNameInput}
+          onChange={(e) => setFirstInput(e.target.value)}
+          isInvalid={isFirstNameMissing}
+        />
+        {isFirstNameMissing && <FormErrorMessage>{`Please enter a first name.`}</FormErrorMessage>}
+      </Box>
+
+      <Box w="50%">
+        <FormLabel htmlFor="last_name">Last Name</FormLabel>
+        <Input
+          id="last_name"
+          type="text"
+          value={lastNameInput}
+          onChange={(e) => setLastInput(e.target.value)}
+          isInvalid={isLastNameMissing}
+        />
+        {isLastNameMissing && <FormErrorMessage>{`Please enter a second name.`}</FormErrorMessage>}
+      </Box>
+    </HStack>
+  );
+};
+
+const ContactForm: React.FC<contactFormProps> = () => {
+  const [emailInput, setEmailInput] = useState('');
```

Comment:
🔧 We should pass a type to useState if possible
```suggestion
  const [emailInput, setEmailInput] = useState<string>('');
```